### ISSUE TITLE
Make main prompts on Condition cards about how they hinder?

### ISSUE THREAD

**Issue by JayLoomis** [astralfrontier/flip-a-card#12]
The present configuration of Condition cards tends toward prompts that describe the condition. It seems like maybe this arose as a way to differentiate the two sides.  What seems more important to me, though, is how the condition limits the fiction. I think this is especially important in the context of people coming up with their own cards on the fly. If you take a condition for your character, as a fellow player, I want to know how that changes the fictional situation for your character moving forward.

_Wanted_/_Pursued_ is, I think, the best example of the current cards as far as this goes. Probably because in that case, defining the threat  also defines the fictional consequences.

I can do a pass of suggestions if you agree that this is a productive way of normalizing the cards.

**Comment by astralfrontier** [comment]
I agree it's a good way to do it, and I'm interested in suggestions.

**Comment by astralfrontier** [comment]
Condition cards updated. Please review.
